peercrawler.py

```diff
@@ -82,6 +82,34 @@ def count_good_peers(self):
     def count_peers(self):
         return len(self.get_peers_copy())
 
+    def listen_incoming(self):
+        with socket.socket(socket.AF_INET6, socket.SOCK_STREAM) as s:
+            s.bind(("::", 7075))
```

Comment:
The port needs to be programmable and it should not default to 7075 because it would typical to want to runinside the same computer as a node.